book/B-embedding-git/sections/libgit2.asc

```diff
@@ -65,12 +62,12 @@ puts "#{commit.author[:name]} <#{commit.author[:email]}>"
 tree = commit.tree
 ----
 
-As you can see, the code is much less cluttered.
-Firstly, Rugged uses exceptions; it can raise things like `ConfigError` or `ObjectError`  to signal error conditions.
-Secondly, there's no explicit freeing of resources, since Ruby is garbage-collected.
-Let's take a look at a slightly more complicated example: crafting a commit from scratch
+Gaya ng makikita mo, ang code ay cluttered na masmababa.
```

Comment:
two words have been combined. please add space character > ``mas mababa``.

Reply:
change to mas mababa